resources/testharness.js

```diff
@@ -1934,9 +2032,33 @@ policies and contribution forms [3].
     };
 
     Tests.prototype.timeout = function() {
+        var cleaningTest = null;
+
         if (this.status.status === null) {
-            this.status.status = this.status.TIMEOUT;
+            forEach(this.tests,
+                    function(test) {
+                        if (test.phase === test.phases.CLEANING) {
+                            cleaningTest = test;
```

Comment:
It looks like `cleaningTest` will be set to the last test in cleanup, but there can actually just be one. Can you add a comment to ensure the reader that this is not an oversight?

Reply:
Sure. I've also tried to think of a way to re-write the code to better express
the expectation, but besides some sort of "assert" statement, I can't think of
anything better.

Reply:
Yeah, I think a comment is the best tradeoff, if we start asserting things and they ever fail, we'd have to think about where those exception end up and if it'll look like a harness error or a test error. No thanks :)